chromium_src/chrome/browser/ui/startup/infobar_utils.cc

```diff
@@ -16,10 +15,8 @@ class BraveGoogleKeysInfoBarDelegate {
 };
 
 #define ShowIfNotOffTheRecordProfile ShowIfNotOffTheRecordProfileBrave
-#define ShowDefaultBrowserPrompt ShowDefaultBraveBrowserPrompt
```

Comment:
is `ShowDefaultBraveBrowserPrompt` used anywhere now?

Reply:
@rebron Should we remove our prompt impls also? Or can be back again later?

Reply:
We should remove. We may go back again later depending on what p3a shows.

Reply:
@goodov Removed by https://github.com/brave/brave-core/pull/15427/commits/786f579d68114f97bc203978f1323d2bfc5d2d7a PTAL. Thanks!